backend/src/libs/packages/config/token.config.package.ts

```diff
@@ -0,0 +1,46 @@
+import convict, { type Config as TConfig } from 'convict';
```

Comment:
you don't need an additional class. just add a new getter `AUTH` to the config class. without ENV parsing for them. just hardcode them